### PR TITLE
improved documentation for component 'Semi-automatic and automatic an…

### PR DESCRIPTION
**Pull request by yanmu2017**
notes...

### Motivation and context

I found some missing steps in the documentation of "Semi-automatic and automatic annotation".
As suggested by @azhavoro in issue isse [2245](https://github.com/openvinotoolkit/cvat/issues/2245), 
I prepared this PR.

### How has this been tested?

I followed the steps in the new documentation on three machines and they all work.

### Checklist

- [ x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat/apps/documentation/installation.md [hunk 1]
```diff
@@ -281,9 +281,24 @@ docker-compose -f docker-compose.yml -f components/analytics/docker-compose.anal
...
+
+- You also have to install `nuctl` command line tool to build and deploy serverless
 functions. Download [the latest release](https://github.com/nuclio/nuclio/releases).
+For example, for Ubuntu you can download the binary file `nuctl-1.4.18-linux-amd64`.
+Put it in the CVAT root directory and change its permission to be executable.
+
+- [Run Nuclio dashboard with](https://github.com/nuclio/nuclio)
+
+```bash
+ docker run -p 8070:8070 -v /var/run/docker.sock:/var/run/docker.sock -v /tmp:/tmp --name nuclio-dashboard quay.io/nuclio/dashboard:stable-amd64
```
Comment (azhavoro) [inline cvat/apps/documentation/installation.md:299]: I think it's unnecessary step, it's already configured here  https://github.com/openvinotoolkit/cvat/blob/develop/components/serverless/docker-compose.serverless.yml#L4-L20

File: cvat/apps/documentation/installation.md [hunk 2]
```diff
@@ -307,7 +322,10 @@ nuctl deploy --project-name cvat \
     --platform local
 ```
 
-Note: see [deploy.sh](/serverless/deploy.sh) script for more examples.
+Note:
+- see [deploy.sh](/serverless/deploy.sh) script for more examples.
+- As an example, if you use `nuctl-1.4.18-linux-amd64` in the CVAT root directory,
+you need to change the `nuctl` in the above commands to `.nuctl-1.4.18-linux-amd64`.
```
Comment (azhavoro) [inline cvat/apps/documentation/installation.md:328]:
```suggestion
you need to change the `nuctl` in the above commands to `nuctl-1.4.18-linux-amd64`.
```

**Comment by azhavoro** [comment]
@yanmu2017 Thank you for the contribution!

Could you fix a couple of minor comments above and fix the linter issues? 
https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=6300831

File: cvat/apps/documentation/installation.md (hunk 2 above)
Comment (nmanovic) [inline cvat/apps/documentation/installation.md:328]: Probably a better approach is to rename nuclt-* to nuctl or make a soft link `ln nuctl-1.4.18-linux-amd64 nuctl`

**Comment by nmanovic** [comment]
No reply from the contributor. I will close the PR.

**Pull request closed without merging**